tests/dash/proto_utils.py

```diff
@@ -66,6 +66,36 @@ def parse_guid(guid_str):
     return {"value": parse_byte_field(uuid.UUID(guid_str).hex)}
 
 
+def parse_range(range_str):
+    parts = range_str.split(",")
+    num_parts = len(parts)
+    if num_parts != 2:
+        raise ValueError("Input string must contain exactly two numbers separated by a comma.")
+    try:
+        int(parts[0])
+        int(parts[1])
+    except ValueError:
+        raise ValueError("Both parts of the input string must be valid integers.")
+    return {"min": parts[0], "max": parts[1]}
+
+
+def parse_value_or_range(value_or_range):
+    if isinstance(value_or_range, int):
+        return {"value": value_or_range}
+    else:
+        parts = value_or_range.split(",")
+        if len(parts) == 1:
+            try:
+                int(parts[0])
+            except ValueError:
+                raise ValueError("Input string must be a valid integer.")
+            return {"value": parts[0]}
```

Comment:
Since we're using an `int()` cast to check validity of the provided value, I think it makes sense to also cast the final returned value.